### PR TITLE
Add pin code session setup example.

### PR DESCRIPTION
**Pull request by Rugvip**

### CODE REVIEW

File: users/pinSession.html [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+                        <button type="submit">Create new room</button>
+                    </li>
+                </ul>
+            </form>
+        </section>
+        <section id="inside-meeting" hidden>
+          <h1>In room with pin <span id="room-pin-label">345</span></h1>
+          Other users in the room:
+          <ul id="member-list">
+              <li>@45645636:webrtc-text.cct.ericsson.net</li>
```
Comment (stefanalund) [inline users/pinSession.html]: Should it be `webrtc-test` and not `webrtc-text`?

**Review by stefanalund — CHANGES_REQUESTED** [review]
Fix typo

File: users/pinSession.html (hunk 1 above)
Comment (Rugvip) [reply users/pinSession.html]: Oops, that was a placeholder, the entire li should be removed

**Review by stefanalund — CHANGES_REQUESTED** [review]
screen shot 2016-09-15 at 15 43 01

Does not work on Safari.

**Comment by Rugvip** [comment]
Ok then, screw security :<

`crypto.getRandomValues()` is supported in IE 11 and Safari, but it's not that important. Pins will be guessable anyway, since it's a pretty small space.

**Review by stefanalund — CHANGES_REQUESTED** [review]
It shouldn't be possible to use non numerical PIN codes, right? I was able to put in chars.

**Comment by stefanalund** [comment]
Furthermore, I think that the 2 buttons should be on the same horizontal line.

**Comment by Rugvip** [comment]
If the browser doesn't support <input type='number'> you can, but that's fine since it doesn't really break things imo. You're supposed to have a specific PIN to enter anyway.

**Comment by Rugvip** [comment]
Changed the create room button to be a bit more clear

**Review by stefanalund — APPROVED** [review]

**Pull request merged**